pkg/columns/columns.go

```diff
@@ -502,6 +502,31 @@ func GetFieldFuncExt[OT any, T any](column ColumnInternals, raw bool) func(entry
 	}
 }
 
+// GetFieldAsArrayFunc returns a helper function to access an array of type OT of a struct T
+// without using reflection. It does not differentiates between direct members of the struct and
+// members of embedded structs.
+func GetFieldAsArrayFunc[OT any, T any](column ColumnInternals) func(entry *T) []OT {
+	l := column.(*Column[T]).Type().Len()
+	s := column.(*Column[T]).Type().Elem().Size()
+
+	return func(entry *T) []OT {
+		r := []OT{}
+		entryStart := unsafe.Pointer(entry)
+		if column.(*Column[T]).getStart != nil {
+			entryStart = column.(*Column[T]).getStart(entry)
+		}
+
+		fieldStart := unsafe.Add(entryStart, column.getOffset())
+
+		for i := 0; i < int(l); i += int(s) {
```

Comment:
I guess since we're doing heavy unsafe magic here already, we could also just copy() the entire thing in one go to the preallocated `r`.

Reply:
Do we actually need to copy it?
If not then something like 
```golang
fieldStart := unsafe.Add(entryStart, column.getOffset())
arrPtr := (*[1 << 30]OT)(fieldStart)
return (*arrPtr)[:l:l]
```
should be better since we are not copying anything

Reply:
It should be fine to not copy it, but is it worth it? Perhaps the copy is not too expensive and we void future headaches?

Reply:
I'm not sure about this, tbh. Afaiu it, generically it should be okay to do something like this (although `unsafe.Slice` would be preferred), however: we don't really know what the source of the memory is and whether it's safe after calling this function. For example, we could be accessing a source that does it's own garbage collection or frees the accessed memory block after some time by other means.